qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/XssOnPhoneNumberFieldSteps.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertTrue;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.helper.TestData;
+import org.openmrs.contrib.qaframework.page.ActiveVisitsPage;
+import org.openmrs.contrib.qaframework.page.RegistrationEditSectionPage;
+
+public class  XssOnPhoneNumberFieldSteps  extends Steps {
+    
+    private TestData.PatientInfo patient;
+    private ActiveVisitsPage activeVisitsPage;
+    private RegistrationEditSectionPage registrationEditSectionPage;
+
+    @Before(RunTest.HOOK.SELENIUM_XSS_VULNERABILITY)
+    public void setUp() throws Exception {
+        initiateWithLogin();
+        patient = createTestPatient();
+        new TestData.TestVisit(patient.uuid, TestData.getAVisitType(),
+         getLocationUuid(homePage)).create();
```

Comment:
```suggestion
        new TestData.TestVisit(patient.uuid, TestData.getAVisitType(),  getLocationUuid(homePage)).create();
```

Reply:
Hey @kdaud  what is the difference here according to your suggestion above.

Reply:
checkout my recent comments.

Reply:
Resolved.